core/lib/default_da_clients/src/object_store/config.rs

```diff
@@ -0,0 +1,14 @@
+use zksync_config::ObjectStoreConfig;
+use zksync_env_config::envy_load;
+
+pub struct ObjectStoreDAConfig {
+    pub config: ObjectStoreConfig,
+}
+
+impl ObjectStoreDAConfig {
+    pub fn from_env() -> anyhow::Result<Self> {
+        Ok(Self {
+            config: envy_load("object_store", "OBJECT_STORE_DA_CLIENT_")?,
```

Comment:
Bikeshedding: IMO, `DA_CLIENT_OBJECT_STORE_` prefix would be more in line with other object store configs (e.g., `PUBLIC_OBJECT_STORE_` or `SNAPSHOTS_OBJECT_STORE_`). For the same reason, it may make sense to name the struct `DAObjectStoreConfig`.

Reply:
makes sense, changed it